### ISSUE TITLE
Uninstall errored because Sequoia upgrade deleted _nixbld1-4

### ISSUE THREAD

**Issue by xaskii** [DeterminateSystems/nix-installer#1220]
## Error
```
Error: 
   0: Error reverting
   0: Action `create_users_and_group` errored
      Multiple child errors

      Action `create_user` errored
      Failed to execute command with status 185 `"/usr/bin/dscl" "." "-delete" "/Users/_nixbld1"`, stdout: delete: Invalid Path

      stderr: <dscl_cmd> DS Error: -14009 (eDSUnknownNodeName)



      Action `create_user` errored
      Failed to execute command with status 185 `"/usr/bin/dscl" "." "-delete" "/Users/_nixbld2"`, stdout: delete: Invalid Path

      stderr: <dscl_cmd> DS Error: -14009 (eDSUnknownNodeName)



      Action `create_user` errored
      Failed to execute command with status 185 `"/usr/bin/dscl" "." "-delete" "/Users/_nixbld3"`, stdout: delete: Invalid Path

      stderr: <dscl_cmd> DS Error: -14009 (eDSUnknownNodeName)



      Action `create_user` errored
      Failed to execute command with status 185 `"/usr/bin/dscl" "." "-delete" "/Users/_nixbld4"`, stdout: delete: Invalid Path

      stderr: <dscl_cmd> DS Error: -14009 (eDSUnknownNodeName)




```

## Metadata
|key|value|
|--|--|
|**version**|0.20.2|
|**os**|macos|
|**arch**|aarch64|

**Comment by cole-h** [comment]
I imagine you updated to macOS Sequoia and tried to uninstall without running the repair tool we released in https://github.com/DeterminateSystems/nix-installer/releases/tag/v0.26.0? If so, then this is more or less expected -- the Sequoia upgrade stole those users from us for its own system purposes.

Did anything else error?

**Comment by cole-h** [comment]
(I'm going to close this as a duplicate of https://github.com/DeterminateSystems/nix-installer/issues/1197, but if you need more help feel free to ask more questions and/or reopen!)

**Comment by xaskii** [comment]
> I imagine you updated to macOS Sequoia and tried to uninstall without running the repair tool we released in [`v0.26.0` (release)](https://github.com/DeterminateSystems/nix-installer/releases/tag/v0.26.0)? If so, then this is more or less expected -- the Sequoia upgrade stole those users from us for its own system purposes.
> 
> Did anything else error?

Yeah, nothing else went wrong, thanks for the response. I'll follow the steps from #1197
